From dae66b04aadfe7ec24b104ebd9bb48f82426205a Mon Sep 17 00:00:00 2001
From: Victor Toso <me@victortoso.com>
Date: Fri, 3 Mar 2017 15:37:01 +0100
Subject: [PATCH 22/26] Fix build when building without oVirt

As remote_viewer_iso_list_dialog_new() is defined on
remote-viewer-iso-list-dialog.h which is only build with oVirt
integration.

 > undefined reference to `remote_viewer_iso_list_dialog_new'

Note that the callback virt_viewer_window_menu_change_cd_activate() is
only visible if oVirt is built in.

Signed-off-by: Victor Toso <victortoso@redhat.com>
---
 src/virt-viewer-window.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/virt-viewer-window.c b/src/virt-viewer-window.c
index 8eda12e..9a0932a 100644
--- a/src/virt-viewer-window.c
+++ b/src/virt-viewer-window.c
@@ -1074,6 +1074,7 @@ void
 virt_viewer_window_menu_change_cd_activate(GtkWidget *menu G_GNUC_UNUSED,
                                            VirtViewerWindow *self)
 {
+#if HAVE_OVIRT
     VirtViewerWindowPrivate *priv = self->priv;
     GtkWidget *dialog;
     GObject *foreign_menu;
@@ -1092,6 +1093,7 @@ virt_viewer_window_menu_change_cd_activate(GtkWidget *menu G_GNUC_UNUSED,
     g_signal_connect(dialog, "response", G_CALLBACK(iso_dialog_response), NULL);
     gtk_widget_show_all(dialog);
     gtk_dialog_run(GTK_DIALOG(dialog));
+#endif
 }
 
 static void
-- 
2.12.0

